CODEOWNERS

```diff
@@ -58,6 +58,7 @@
 /pkg/identityserver @htdvisser @bafonins @kschiffer
 /pkg/email @htdvisser @bafonins @kschiffer
 /pkg/oauth @htdvisser @bafonins @kschiffer
+/pkg/account @htdvisser @bafonins @kschiffer
```

Comment:
This is my only code owned file and I'm fine with this if this was discussed before.

Maybe loop in @asmulko or @mjamescompton instead of @htdvisser to reduce his scope

Reply:
We should indeed take a better look at the packages that are code-owned by me. In this case it's fine to add me me. (1) It's pretty much all code that was moved from `pkg/oauth`, so I already owned it. (2) It's pretty small and (3) I don't expect it to change often.